pkg/cmd/pulumi/install.go

```diff
@@ -114,60 +106,15 @@ func newInstallCmd() *cobra.Command {
 
 			if !noPlugins {
 				// Compute the set of plugins the current project needs.
-				installs, err := lang.GetRequiredPlugins(programInfo)
+				plugins, err := lang.GetRequiredPlugins(programInfo)
 				if err != nil {
 					return err
 				}
 
-				// Now for each kind, name, version pair, download it from the release website, and install it.
-				for _, install := range installs {
-					// PluginSpec.String() just returns the name and version, we want the kind too.
-					label := fmt.Sprintf("%s plugin %s", install.Kind, install)
```

Comment:
> we want the kind too

I'm not sure why we care about the kind. I'm pretty sure in all cases we only care about `apitype.ResourcePlugin` --- that's the kind of plugin that will be returned from `lang.GetRequiredPlugins`. But, just pointing out that  `engine.EnsurePluginsAreInstalled` doesn't show the kind in the label, aside from the new `!reinstall` path. I _think_ this is likely OK, but would be good for @Frassle to comment about the intent here.

Reply:
It's probably fine for now. I could imagine a more flexible future might have other plugin kinds returned here but we can worry about that later.